errors/errors_task.go

```diff
@@ -142,17 +142,33 @@ func (err *TaskCancelledNoTerminalError) Code() int {
 }
 
 // TaskMissingRequiredVars is returned when a task is missing required variables.
+
+type MissingVar struct {
+	Name          string
+	AllowedValues []string
+}
 type TaskMissingRequiredVars struct {
```

Comment:
Can we add the `Error` suffix to this type as per the [go naming conventions for errors]( https://go.dev/wiki/Errors#naming). Same with `TaskNotAllowedVars` below.

Reply:
Yeah good catch
Only for `TaskMissingRequiredVars` right ? MissingVar is not the error but the underlying details

Reply:
Yeah exactly :) Thanks!